src/releng_docs/services_shipit.rst

```diff
@@ -56,4 +56,61 @@ It does the following tasks on every run:
 - Report the full analysis to shipit dashboard, so it can be displayed on shipit frontend.
 
 
+``shipit v1 and v2 interaction``
+--------------------------------
+
+Initially, shipit-v2 will compliment v1 by adding features, as opposed to replacing. The following is a implementation strategy for how that will work.
+
+**ship-it v1 will solely be responsible for submitting and kicking off a new release**
+
+.. code-block:: bash
+
+    % curl -X POST -d "product=firefox&version=99.0" https://ship-it.mozilla.org/submit_release.html
+
+releaserunner_ continues to behave as normal by polling shipit v1 for new releases and, when found, runs release_sanity_, creates taskcluster_graph1_, and notifies release-drivers of new release.
+In addition, releaserunner will tell shipit-v2 about the new release and ask shipit-v2 to create a pipeline:
+
+.. code-block:: python
+
+    r = requests.post("https://mozilla-releng.net/shipit_pipeline", data={uid='foo', pipeline={}})
+
+shipit-v2's pipeline will consist of steps representing the release. A step could be sign off step (shipit_signoff), or a taskcluster step (shipit_taskcluster).
```

Comment:
This reads as if shipit_signoff and shipit_taskcluster are the only available steps. Rewording this slightly might make it hold up better over time.